.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           python-version: "3.7.x"
           architecture: "x64"
-      - run: pip install pytest pytest-cov pydocstyle numpydoc
+      - run: pip install pytest pytest-cov pytest-xdist pydocstyle numpydoc
```

Comment:
I don't think we need `xdist` here because this is about checking docs, they're fast enough without that :)